accounts/usbwallet/ledger.go

```diff
@@ -162,8 +162,8 @@ func (w *ledgerDriver) SignTx(path accounts.DerivationPath, tx *types.Transactio
 		return common.Address{}, nil, accounts.ErrWalletClosed
 	}
 	// Ensure the wallet is capable of signing the given transaction
-	if chainID != nil && w.version[0] <= 1 && w.version[1] <= 0 && w.version[2] <= 2 {
-		return common.Address{}, nil, fmt.Errorf("Ledger v%d.%d.%d doesn't support signing this transaction, please update to v1.0.3 at least", w.version[0], w.version[1], w.version[2])
+	if chainID != nil && w.version[0] <= 1 && w.version[2] <= 2 {
```

Comment:
Why did you remove the version[1] check?

Reply:
Because `staticcheck` complains that a `byte` is always greater than 0.

Reply:
Ah, good point :P